src/styles/layouts/_layout-authentication.scss

```diff
@@ -18,7 +18,12 @@
   }
 
   @include respond-above('xl') {
+    grid-template-columns: 1fr minmax(rem(1104px), rem(1440px)) 1fr;
```

Comment:
Currently the background image is limited to `width` of `1440px`. It should cover all the `main`. Try using mobile `grid-template-areas` for every breakpoint and set the **max** of the `form` at `sign-in-form`.